dev_tools/simulate_github_lambda/simulate-github-lambda.py

```
@@ -0,0 +1,199 @@
+from __future__ import annotations
+
+import json
+import os
+from functools import cache
+from unittest.mock import patch
+
+import boto3
+import yaml
+
+from iambic.core.logger import log
+from iambic.plugins.v0_1_0.github.github_app import run_handler
+
+DEV_REGION = os.environ.get("DEV_REGION", "us-west-2")
+DEV_EMAIL_DOMAIN_SUFFIX = os.environ.get("DEV_EMAIL_DOMAIN_SUFFIX", "@example.com")
+DEV_ACCOUNT_ID = os.environ.get("DEV_ACCOUNT_ID", "")
```

Should we assert the last two and raise if not defined?